### ISSUE TITLE
Content-Disposition, Content-Type, name, filename appear in the file's body after uploading to the s3 bucket using presigned URL

### ISSUE THREAD

**Issue by ngohuytrieu** [aws/aws-sdk-php#2510]
### Describe the bug

I'm attempting to use presigned S3 URLs to PUT/GET a file via requests but getting additional information in the uploaded file:
`Hello world!`

The uploaded file on S3 looks like this:
```
----------------------------375723512835085641445297
Content-Disposition: form-data; name=""; filename="helloworld.txt"
Content-Type: text/plain

Hello world!
----------------------------375723512835085641445297--
```

Is there any reason that information would be appearing in the uploaded file?

### Expected Behavior

The uploaded file remaining the origin file's content

### Current Behavior

The file's content has been changed

### Reproduction Steps

1. Generate pre-signed URL
2. Call pre-signed URL using POSTMAN alternative tool

### Additional Information/Context

Here is my PHP code generating pre-signed URL:
```php
$s3Client = S3Client::factory(array(
    'credentials' => array(
        'key'    => '<my_key>',
        'secret' => '<my_secret>',
    ),
    'region' => '<my_region>,
    'version' => '2006-03-01'
));
$cmd = $s3Client->getCommand('PutObject', [
    'Bucket' => "<my_bucket>",
    'Key'    => "helloworld.txt",
    // 'Body'   => "BODY",
    'ContentType' => "text/plain"
]);

$request = $s3Client->createPresignedRequest($cmd, '+60 minutes');
$presignedUrl = (string) $request->getUri();
echo $presignedUrl;
```

I used POSTMAN alternative tool to call up the repsigned URL to upload the file
<img width="706" alt="image" src="https://user-images.githubusercontent.com/54321995/186106695-312b60a2-7b3f-4277-bb02-f25208bf38e2.png">

I also tried to use axios to call up the pre-signed URL but the result is still the same
```javascript
const url = 'my_presigned_url'

const formData = new FormData()
formData.append('file', file_data)

axios(url, {
  method: 'put',
  headers: { 'Content-Type': 'multipart/form-data' },
  body: formData
})
  .then((response) => {
    console.log({ aws: response })
  })
  .catch((error) => {
    console.error({ awserror: error })
  })
```
### SDK version used

3

### Environment details (Version of PHP (`php -v`)? OS name and version, etc.)

Macos

**Comment by ngohuytrieu** [comment]
@yenfryherrerafeliz thank you for replying.

So basically, I have one client app, and that client app needs to upload a file to amazon S3

But the app shouldn't handle the Amazone credentials so I create an API generating the presigned URL then the client app calls up to the presigned URL to upload file to amazone S3

**Here are the working flow steps by step:**
1. The server create an API to generate presigned URL
2. The client app calls API from "**1.**" to get presigned URL
3. The client app calls up to the presigned URL from "**2.**" with the file's content inside the body of the request to upload file to amazon S3

The content of the file is attached from the client app, I don't set the content of the file by command. The request code from client app may look like this:
```javascript
const url = 'my_presigned_url'

const formData = new FormData()
formData.append('file', file_data)

axios(url, {
  method: 'put',
  headers: { 'Content-Type': 'multipart/form-data' },
  body: formData
})
  .then((response) => {
    console.log({ aws: response })
  })
  .catch((error) => {
    console.error({ awserror: error })
  })
```

The problem is when the client calls up to the presigned URL with the file's content in the body like:
```
Hello world!
```

But when I open the file on amazon s3 bucket, the content of the file looks like this:
```
----------------------------375723512835085641445297
Content-Disposition: form-data; name=""; filename="helloworld.txt"
Content-Type: text/plain

Hello world!
----------------------------375723512835085641445297--
```

**Comment by yenfryherrerafeliz** [comment]
@ngohuytrieu I do not know why or how you are getting that data appended to your uploaded file. As I mentioned, your file content will be whatever valid data that is set as the body of the request. However, could you please try the following on your front end logic:
```js
const url = 'my_presigned_url'

axios(url, {
  method: 'put',
  data: file_data
}).then((response) => {
    console.log({ aws: response })
}).catch((error) => {
    console.error({ awserror: error })
});
```
I did multiple tests by using the example above and I got successfully results, so please let me know if works for you.

Could you please also try to put a file without a presigned url?, as follow:
```php
$s3Client = S3Client::factory([
    'region' => 'us-east-2',
    'version' => '2006-03-01'
]);
$response = $s3Client->putObject([
    'Bucket' => 'herrergy-sample-bucket',
    'Key' => 'my-new-key.txt',
    'Body' => 'here is a test body!'
]);
```
Did you get the same behavior?, if not, then the problem may be when executing the request from the front end implementation.

Thanks!

**Comment by ngohuytrieu** [comment]
Thank @yenfryherrerafeliz for your reply,

It works now. I used your code and it works
```javascript
const url = 'my_presigned_url'

axios(url, {
  method: 'put',
  data: file_data
}).then((response) => {
    console.log({ aws: response })
}).catch((error) => {
    console.error({ awserror: error })
});
```

Turns out if I use POSTMAN, the body if I set form data type then that issue occurs.
Just switch to text data type in the body then it works. For other files types like image or video still don't know how to run that with POTMAN but the node js script works well

Thank you so much for your help!
